### PR TITLE
Introduce commit

### PR DESCRIPTION
**Pull request by gigbat**

### CODE REVIEW

File: src/main/java/com/dev/cinema/model/MovieSession.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+public class MovieSession {
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn (name = "movie_id")
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDate localDate;
```
Comment (yurwing) [inline src/main/java/com/dev/cinema/model/MovieSession.java]: You have to use here LocalDateTime

**Review by yurwing — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/model/MovieSession.java (hunk 1 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/MovieSession.java]: done

**Review by yurwing — APPROVED** [review]

File: src/main/java/com/dev/cinema/model/MovieSession.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table (name = "movieSession")
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dev/cinema/model/MovieSession.java]:
```suggestion
@Table (name = "movie_session")
```
normally we are using underscore style for table names

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job

File: src/main/java/com/dev/cinema/model/MovieSession.java (hunk 2 above)
Comment (gigbat) [reply src/main/java/com/dev/cinema/model/MovieSession.java]: done

**Pull request merged**